packages/serverpod_client/test/exception_from_test.dart

```diff
@@ -0,0 +1,183 @@
+import 'dart:convert';
+
+import 'package:serverpod_client/serverpod_client.dart';
+import 'package:serverpod_client/src/serverpod_client_shared_private.dart';
+import 'package:test/test.dart';
+
+class TestSerialization extends SerializationManager {}
+
+class TestSerializationImpl extends SerializationManager {
+  @override
+  Object? decodeWithType(String data) {
+    var json = jsonDecode(data);
+
+    if (json['className'] == 'TestException') {
+      return TestException.fromJson(json['data']);
+    }
+
+    throw Exception('Unknown exception type');
+  }
+}
+
+class TestException extends SerializableException {
+  String message;
+
+  TestException(this.message);
+
+  factory TestException.fromJson(Map<String, dynamic> json) {
+    return TestException(json['message']);
+  }
+
+  @override
+  dynamic toJson() {
+    return {
+      'message': message,
+    };
+  }
+}
+
+void main() {
+  group('Given a serializable exception when parsing the exception', () {
+    var exception = getExceptionFrom(
+      data:
+          '{"className": "TestException", "data": {"message": "A custom exception"}}',
+      serializationManager: TestSerializationImpl(),
+      statusCode: 400,
+    );
+
+    test('then a base ServerpodClientException is created', () {
```

Comment:
chore: Update description so that it matches the test.

Reply:
This applies to then next test as well so please check all tests in this file so the descriptions match.